documentation/general/dash-sonic-hld.md

```diff
@@ -374,6 +388,8 @@ underlay_ip              = ip_address                ; PA address for the CA
 mac_address              = MAC address as string     ; Inner dst mac
 metering_bucket          = bucket_id                 ; metering and counter
 use_dst_vni              = bool                      ; if true, use the destination VNET VNI for encap. If false or not specified, use source VNET's VNI
+overlay_sip              = ip_address                ; overlay src ip if routing_type is {privatelink}, transform last 32 bits from packet 
```

Comment:
How does this interact with `pl_sip_encoding` from the ENI table above?

Reply:
pl_sip_encoding should override the bits in here